forms.rst

```diff
@@ -345,13 +345,13 @@ can set this option to generate forms compatible with the Bootstrap 4 CSS framew
     .. code-block:: php
 
         // config/packages/twig.php
-        $container->loadFromExtension('twig', [
-            'form_themes' => [
-                'bootstrap_4_layout.html.twig',
-            ],
+        use Symfony\Config\TwigConfig;
+
+        return static function (TwigConfig $twig) {
+            $twig->formTheme(['bootstrap_4_layout.html.twig']);
```

Comment:
Just asking: is this `form_themes` ---> `->formTheme()` transformation (from plural to singular) intended? Because when we pass an array as argument, the method is always named in plural, so it should be `->formThemes()`

Reply:
I dug into this. You are correct. We can be sure that this should always be an array and we should not make the method name singular. 

See https://github.com/symfony/symfony/pull/40878

Reply:
I assume PR symfony/symfony#40878 will be merged. I've updated the docs